tests/components/homekit/test_accessories.py

```diff
@@ -2,6 +2,7 @@
 
 This includes tests for all mock object types.
 """
+# pylint: skip-file
```

Comment:
This one

Reply:
Oof. Good catch. My requirements were out of date. :flushed: